docs/tutorial/part03.rst

```diff
@@ -2,5 +2,250 @@
 How to create the base models.
 ==============================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+Create a new app for the API: 
+==============================
+Right now, we need to create an app which is going to be use  to create the API and, with it, the models. If you do not know the definition of an API; it is “`code that allows two software programs to communicate with each other`”. 

```

Comment:
Hey in here
Is `is going to be use` or `is going to be used`? 🤔

Reply:
I believe both are correct.

Reply:
👌  is more for my education. Thanks for the answer 😅